manager/manager.go

```diff
@@ -49,6 +49,16 @@ func (man *volumeManager) doCreate(volume *types.VolumeInfo) (*types.VolumeInfo,
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to create volume '%s'", volume.Name)
 	}
+
+	if len(vol.Replicas) == 0 {
```

Comment:
Not quite sure. The create volume logic in cattle will return a volume if it already exists. See https://github.com/rancher/longhorn-orc/blob/master/orch/cattle/cattle.go#L179

I don't know in which situation this will happen, probably from recovery? But if it happens, I don't think we should create x more replicas.